docs/operating-scylla/procedures/cluster-management/create-cluster-multidc.rst

```diff
@@ -209,6 +209,19 @@ In this example, we will show how to install a nine nodes cluster.
    UN   54.187.142.201  109.54 KB       256     ?               d99967d6-987c-4a54-829d-86d1b921470f    RACK1
    UN   54.187.168.20   109.54 KB       256     ?               2329c2e0-64e1-41dc-8202-74403a40f851    RACK1
 
-See also:
+--------------------------
+Preventing Quorum Loss
```

Comment:
The text here is similar to arbiter-dc.rst above.
Let's link to it instead of repeating the content (and my comments ;)